src/gateway_config_worker.erl

```diff
@@ -158,3 +159,14 @@ maybe_signal_position(#state{gps_lock=true}) ->
     self() ! signal_position;
 maybe_signal_position(_) ->
     ok.
+
+
+-spec report_gps_stats(ubx:nav_pvt()) -> ok.
+report_gps_stats(NavMap) when map_size(NavMap) == 0 ->
+    ok;
+report_gps_stats(#{num_sats := NumSats, t_acc := TimeAcc, h_acc := HorizontalAcc, fix_type := FixType}) ->
+    dogstatsd:gauge([
+                     {"gps.num_sats", NumSats, #{fix_type => FixType}},
+                     {"gps.t_acc", TimeAcc, #{fix_type => FixType}},
```

Comment:
Why is fix_type metadata, rather than an independent value?

Reply:
I was assuming that the user would want to filter out the data using the fix_type tag
so that you could see the readings at various fixes. 

And more importantly filter out/recognize fix type 0 readings

Reply:
I may well have misunderstood how we'd want to use and slice up the data

Reply:
I have no idea at all, I was just curious.